core/src/main/scala/org/apache/spark/scheduler/ReplayListenerBus.scala

```diff
@@ -17,6 +17,7 @@
 
 package org.apache.spark.scheduler
 
+import com.fasterxml.jackson.core.JsonParseException
```

Comment:
The import order is wrong. See https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide#SparkCodeStyleGuide-Imports